tests/help.rs

```diff
@@ -1524,12 +1522,12 @@ fn show_short_about_issue_897() {
 #[test]
 fn issue_1487() {
     let app = App::new("test")
-        .arg(Arg::with_name("arg1")
-            .group("group1"))
-        .arg(Arg::with_name("arg2")
-            .group("group1"))
-        .group(ArgGroup::with_name("group1")
-            .args(&["arg1", "arg2"])
-            .required(true));
+        .arg(Arg::with_name("arg1").group("group1"))
+        .arg(Arg::with_name("arg2").group("group1"))
+        .group(
+            ArgGroup::with_name("group1")
+                .args(&["arg1", "arg2"])
+                .required(true),
+        );
```

Comment:
This one is unfortunate since the indentation was on purpose. Maybe throw `#[rustfmt::skip]` in?

Reply:
done!